pub/docs/js.md

```diff
@@ -1,74 +1,251 @@
+### [Literal: Regular Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)
+
+* _symbol_: ``
+* _pronunciation_:
+* _examples_:
+```javascript
+
+```
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Operator: Modulus](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+* _symbol_: `%`
+* _pronuntciation_: 'mod u lus'
+* _examples_:
+```javascript
+12 % 5 // 2 and 15 % 3 // 0
+```
+
+### [Operator: Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+* _symbol_: `++`
+* _pronunciation_: `in cre ment`
+* _examples:
+```javascript
+4++ // 5, 6, 7....
+```
+
+### [Operator: Equal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `==`
+* _pronunciation_: "equals"
+* _examples_: 
+```javascript
+4 == "four"  // false  and 4 == 5 // false
+```
+
+### [Operator: Not Equal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `!=`
+* _pronunciation_: does not equal
+* _examples: 
+```javascript
+4 != 4 // false and 4 != 5 // true
+```
+### [Operator: Strict Not Equal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `!==`
+* _pronunciation_: "strict not equals"
+* _example_:
+```javascript
+var1 !== ""3" and 3 !== '3'
+```
+
+### [Logical Operator: &&](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `&&`
+* _pronunciation_: Logical AND
+* _examples_:
+```javaascript
+var a1 =  true && true;  // t && t returns true and var a5 = "Cat" && "Dog";    // t && t returns Dog
+```
+
+### [Logical Operator: !](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `!`
+* _pronunciation_: `logical not`
+* _examples_:
+```javascript
+var n1 = !true;  // !t returns false and var n2 = !false; // !f returns true
+```
+
+### [String Operator: Concatenation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `+`
+* _pronunciation_: "concatenate"
+* _examples_:
+```javescript
+"cat + "dog" // "catdog" and "10" + 2 // 102
+```
+
+### [Global Object: Undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)
+
+* _symbol_: `undefined`
+* _pronunciation_: "undefined"
+* _examples_: 
+```javascript
+if var(x)=undefined{this code executes} else {this happens}
+```
```

Comment:
Try running this through the REPL or `node` interpreter. JavaScript will not like you... Make sure your examples are real!